deps/src/main/java/com/microsoft/azure/sdk/iot/deps/convention/WritablePropertyResponse.java

```diff
@@ -0,0 +1,19 @@
+package com.microsoft.azure.sdk.iot.deps.convention;
+
+import lombok.Getter;
+import lombok.Setter;
+
+public interface WritablePropertyResponse
+{
+    Object getValue();
```

Comment:
not using lombok getters and setters?

Reply:
I am for implementations., however getter/setters don't work on the interface and you need to explicitly add these.